model/src/main/java/com/cloudera/nav/sdk/model/entities/Entity.java

```diff
@@ -344,4 +351,10 @@ public boolean equals(Object o) {
   public int hashCode() {
     return identity.hashCode();
   }
+
+  public Boolean getIsEndPoint() { return isEndPoint; }
+
+  public void setIsEndPoint(Boolean isEndPoint) {
```

Comment:
It is similar to the user having to create an EndPointProxy. Instead of the user having to create that EndPointProxy, the user can simply create the type of Entity they want to link to and set this flag.